video/vdu_buffered.h

```diff
@@ -253,7 +253,7 @@ void IRAM_ATTR VDUStreamProcessor::vdu_sys_buffered() {
 			}
 			auto matrixSize = getMatrixSize(bufferId);
 			if (matrixSize.value != 0) {
-				float transform[matrixSize.size()] = {0.0f};
+				float transform[matrixSize.size()];
```

Comment:
I'm kinda curious as to why the defaults in this file for transform matrixes here got removed

I'm not _entirely_ certain that this is completely safe - I think that some of the functionality inside `bufferAffineTransform` and `bufferMatrixManipulate` for instance is not explicitly setting all values inside a transform matrix.  this could then result in a matrix that has junk values within it, resulting in unexpected behaviour